exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/Metadata.java

```diff
@@ -494,5 +497,48 @@ public ColumnMetadata(SchemaPath name, PrimitiveTypeName primitiveType, Original
       this.min = min;
       this.nulls = nulls;
     }
+
+    @JsonProperty(value = "min")
+    public Object getMin() {
+      if (primitiveType == PrimitiveTypeName.BINARY) {
+        if (originalType == OriginalType.UTF8) {
```

Comment:
Had an offline discussion with Aman, we can remove this check and serialize everything using getBytes()